docs/design/alias-configuration.md

```diff
@@ -52,4 +69,56 @@ In this flow
 
 ## Open Questions
 
-## Acceptance Tests
\ No newline at end of file
+## Acceptance Tests
+
+## Cases
+
+- Crypto Transfer Transaction to EVM address alias should create an account with its alias field set to the EVM address and its key will be empty (to be populated by a signed transaction as part of lazy account create)
+- Crypto Transfer Transaction (without auto creation): to be called with EVM address (the account with this address is already auto lazy created)
+  - We need to extract the public key from signature with extractSig
+  - After that to use the extracted key for recovering the public address (recoverAddressFromPubKey)
+  - Recovered address must match with the alias (public address of the eth account)
+  - If the recovered address match with the address stored in the alias we have to update the key property of the account and store it in the ledger
+  - With the above step the process of lazy creation will be finished
+- Crypto Transfer Transaction to ECDSA key alias should create an account with its alias field set to the ECDSA key, populate the alias map with the EVM address calculated based on the ECDSA key, and the key field set with the value of the ECDSA key (currently implemented) and another Crypto Transfer Transaction to the EVM address corresponding to the ECDSA key should transfer to the account created in the previous transaction
+- Crypto Create Transaction with ED25519 key alias should create an account with its alias and key fields set to the ED25519 key. Attempting another Crypto Create Transaction with the same ED25519 key alias should fail
```

Comment:
Remove this

Reply:
See https://github.com/hashgraph/hedera-services/pull/3943#discussion_r972928819.

Reply:
Removed.